app/javascript/react/components/molecules/CalendarStationHeader/CalendarStationHeader.tsx

```diff
@@ -0,0 +1,141 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+import { ActionButton } from "../../ActionButton/ActionButton.style";
+import { Button } from "../../Button/Button";
+import { StationValueLabel } from "../StationValueLabel";
+import * as S from "./CalendarStationHeader.style";
+import bellAlert from "../../../assets/icons/bellAlert.svg";
+import shareLink from "../../../assets/icons/shareLink.svg";
+import downloadImage from "../../../assets/icons/download.svg";
+import copyLink from "../../../assets/icons/copyLink.svg";
+
+interface CalendarStation {
+  stationName: string;
+  profile: string;
+  sensor: string;
+  lastUpdate: string;
+  streamData: StreamData;
+}
+
+interface StreamData {
+  day: string;
+  value: number;
+  parameter: string;
+}
+
+const CalendarHeaderName: React.FC<{ stationName: string }> = ({
+  stationName,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <S.HorizontalContainer>
+        <S.Description>{t("calendarHeader.stationPrefix")}</S.Description>
+        <S.Header>{stationName}</S.Header>
+      </S.HorizontalContainer>
+    </>
+  );
+};
+
+const CalendarStationProfileSensor: React.FC<{
+  profile: string;
+  sensor: string;
+}> = ({ profile, sensor }) => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <S.HorizontalContainer>
+        <S.RowContainer>
+          <S.Subtitle>
+            {t("calendarHeader.profile")}
+          </S.Subtitle>
+          <S.DataDescription>{profile}</S.DataDescription>
+        </S.RowContainer>
+
+        <S.RowContainer>
+          <S.Subtitle>
+            {t("calendarHeader.sensor")}
+          </S.Subtitle>
+          <S.DataDescription>{sensor}</S.DataDescription>
+        </S.RowContainer>
+      </S.HorizontalContainer>
+    </>
+  );
+};
+
+const CalendarUpdateOccurance: React.FC<{ lastUpdate: string }> = ({
+  lastUpdate,
+}) => {
+  const { t, i18n } = useTranslation();
+
+  return (
+    <S.HorizontalContainer>
+      <S.RowContainer>
+        <S.Subtitle>
+          {t("calendarHeader.updateFrequencyTitle")}
+        </S.Subtitle>
+        <S.FrequencyLabel>
+          {i18n.t("calendarHeader.updateFrequencyValue", { value: 15 })}
```

Comment:
Can't you just use `{t("calendarHeader.updateFrequencyValue", { value: 15 })}`? Than the i18n explicite import would not be needed.